Kernel/FileSystem/Plan9FS/FileSystem.cpp

```diff
@@ -328,6 +328,7 @@ void Plan9FS::thread_main()
 {
     dbgln("Plan9FS: Thread running");
     do {
+        EXIT_KERNEL_TASK_IF_NECESSARY()
```

Comment:
Why is this part of this commit?

Reply:
It's very possible I messed something up; I peeled apart the changes manually this afternoon.